profiler/data_collector.rst

```diff
@@ -268,7 +267,6 @@ to specify a tag that contains the template:
 
         $container
             ->autowire(RequestCollector::class)
-            ->setPublic(false)
```

Comment:
Here too

Reply:
What do you call php-dsl ? The usage of ContainerConfigurator ?